public-react/src/services/keycloak.js

```diff
@@ -0,0 +1,36 @@
+import Keycloak from 'keycloak-js';
+
+const _kc = new Keycloak({
+  url: 'https://dev.loginproxy.gov.bc.ca/auth',
+  realm: process.env.REACT_APP_REALM,
+  clientId: process.env.REACT_APP_CLIENT_ID,
+});
+
+const loginOptions = {
+  redirectUri: `https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=${encodeURIComponent(
```

Comment:
can we add a comment on what this extra URL `https://logon7.gov.bc.ca` does in the codebase for them?